README.md

```diff
@@ -15,19 +15,19 @@ ODBC or ADO.NET, which are mostly old and static standards and create
 additional complexity due to the necessary driver managers.
 
 ## Changes
+* [WebSocket API v4](docs/WebsocketAPIV4.md) support has been added in Exasol 8.31.0.
```

Comment:
If DB Exa to Exa changes are merged this week, then DB version will be 8.32.0.

Reply:
Done